examples/complete/main.tf

```diff
@@ -22,6 +22,7 @@ module "event_streams" {
   tags              = var.resource_tags
   access_tags       = var.access_tags
   topics            = var.topics
+  metrics           = []
```

Comment:
can you add the allowed values here so they get tested in the tests we run?

Reply:
`examples/complete` doesn't set the plan to `enterprise-3nodes-2tb`, so is using a standard plan, and these options are not available for standard.  Checking our provision code, I see that they're just ignored, and don't cause an error - but I can't promise it will always work that way. 

So I could include the allowed values, but it would only be a test of the terraform validator, and might break in the future if we improve the parameters checking.

Reply:
lets not set any metrics in the complete example then. Just add it in the fscloud example